sass/mixins/_mixins.scss

```diff
@@ -0,0 +1,38 @@
+@mixin inner_padding {
+  padding-left: $padding-LR;
+  padding-right: $padding-LR;
+}
+
+@mixin btn_big {
+  display: inline-block;
+  height: 5.8rem;
+  width: 100%;
+  color: $color-white;
+  font-size: $font-size-two;
+  line-height: 5.8rem;
+  text-align: center;
+  text-transform: uppercase;
+  background-color: $color-blue;
+  border: none;
+
+  &:hover {
+    background-color: $color-blue-dark;
+  }
+
+  &:active {
+    color: $color-white-dimm;
+    background-color: $color-blue-darker;
+  }
+
+  &:disabled {
+    background-color: $color-gray;
+  }
+}
+
+@mixin header_one ($size) {
```

Comment:
Why is it a header one?  not two? and it accepts a font-size which could be anything. Would just a header be a better name?